contrib/containerlab/bgpv2/multi-homing/Makefile

```diff
@@ -0,0 +1,20 @@
+deploy:
+	kind create cluster --config cluster.yaml
+	sudo containerlab -t topo.yaml deploy
+	# create secret for bgp
+	kubectl -n kube-system create secret generic --type=string bgp-auth-secret --from-literal=password=cilium123
+	# install cilium
+	helm install cilium -n kube-system oci://quay.io/cilium-charts-dev/cilium --version 1.16.0-dev-dev.247-main-8c97a2119d -f values.yaml
```

Comment:
similarly here, this is a very specific cilium version

Reply:
Made this configurable, waiting for next official release at which point I will remove it and get latest.

Reply:
How did you make it configurable? I don't see the knob. Am I missing something?

Reply:
woops, commit got lost somewhere  :( checking

Reply:
Pushed the change again to have it fixed.